src/include/denc.h

```diff
@@ -1865,7 +1866,7 @@ inline std::enable_if_t<traits::supported && !traits::featured> decode_nohead(
   __u8 struct_compat = compat;						\
   char *_denc_pchar;							\
   uint32_t _denc_u32;							\
-  static_assert(CEPH_RELEASE >= 21 || compat == 2);			\
+  static_assert(CEPH_RELEASE >= (19/*squid*/ + 2) || compat == 2);	\
```

Comment:
`common/ceph_releases.h` has the helpers:

```cpp
// the C++ version of CEPH_RELEASE_* defined by include/rados.h
enum class ceph_release_t : std::uint8_t {
  unknown = 0,
  argonaut,
  bobtail,
  cuttlefish,
  dumpling,
  emperor,
  firefly,
  giant,
  hammer,
  infernalis,
  jewel,
  kraken,
  luminous,
  mimic,
  nautilus,
  octopus,
  pacific,
  quincy,
  reef,
  squid,
  max,
};
```

Anyway, this comment is less-than-nit.